pages/api/bitbucket/callbacks/webhook.ts

```diff
@@ -36,9 +50,12 @@ const webhookHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 		repo_name: repo_name
 	}).catch((error) => {
 		console.error('[webookHandler] Failed to get repoConfig from db :', error);
+		rudderStackEvents.track("absent", "", 'bitbucket-webhook', { type: 'get-repo-config', eventStatusFlag: 0, event_properties });
```

Comment:
Not needed, line 57-58 should be enough. This would result in sending similar events twice